### DISCUSSION TITLE
Why can't I get a nice 'mask_phases'?

### DISCUSSION THREAD

**Discussion by Yzxastr** [phoebe-project/phoebe2#656] [Questions]
I can't get a nice 'mask_phases' when I use 'lc_geometry' to calculate the phase range of eclipse .
Here is my results
![my plot](https://user-images.githubusercontent.com/102363557/200315411-65bc380b-512f-4004-bf2d-7dd3e496dadf.png)
)

**Comment by kecnry** [comment]
The [LC geometry estimator](http://phoebe-project.org/docs/2.4/api/phoebe.parameters.solver.estimator.lc_geometry) is designed for systems that show eclipses, by finding the times of ingress and egress.  It looks here like you have an ellipsoidal variable, so this estimator likely won't give you great results.  Since the estimators just take the observations as input and can't assume anything from the currently set parameters, there isn't really any way for them to detect that this is the case, so it's up to you to interpret whether the proposed results are reasonable (and accept them), or not reasonable (and reject them).

If you're only using the estimator to create the mask, you can instead create it manually by setting the `mask_phases` parameter directly - see the [mask phases tutorial](http://phoebe-project.org/docs/latest/tutorials/mask_phases) for details.
